src/main/resources/com/google/api/codegen/go/example.snip

```diff
@@ -1,86 +1,87 @@
 @extends "go/header.snip"
 
-@snippet generateFilename(service)
-    {@context.getApiConfig.getPackageName}/{@context.getReducedServiceName(service)}_client_example_test.go
-@end
-
-@snippet generateClass(service, body)
+@snippet generate(view)
     {@headerComment()}
 
-    // AUTO-GENERATED CODE. DO NOT EDIT.
-
-    package {@context.getPackageName}_test
+    package {@view.localPackageName}_test
```

Comment:
As mentioned in one of my other comments, I don't know that using the same view for library and sample is the right way to go. 

I think using string concatenation here is just a local optimum. It makes this case different than others because it doesn't follow the convention, making it potentially a pain for people trying to find things based on those conventions.